server/metrics.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	registerations = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Minor naming issue. Some of these metrics like `errorsCounter` have a type of a metrics as a suffix (`Counter`) while others do not have it, for example `registrations`. Wouldn't it be clearer if it was named `registrationsGauge`?